### PR TITLE
React-React1-Week1

### PR DESCRIPTION
**Pull request by dmytriigithub**

### CODE REVIEW

File: react/react1/week1/Todo/src/components/TodoList/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import TodoListItem from "../TodoListItem/TodoListItem"
+import './TodoList.css'
+
```
Comment (LucasIversen) [inline react/react1/week1/Todo/src/components/TodoList/TodoList.jsx:3]: I can see that you've specified which props you want in your components. This is perfectly fine, and will help you make more stable components, especially when working in teams.

File: react/react1/week1/Todo/src/components/TodoList/TodoList.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const TodoList = ({todo}) => {
+  const elements = todo.map(item => {
+    const {id, ...itemProps} = item;
```
Comment (LucasIversen) [inline react/react1/week1/Todo/src/components/TodoList/TodoList.jsx:6]: really good use of the … operator 🥇 Good to see you use the map function, which makes this component much more scalable.

File: react/react1/week1/Todo/src/components/TodoListItem/TodoListItem.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import './TodoListItem.css'
+
```
Comment (LucasIversen) [inline react/react1/week1/Todo/src/components/TodoListItem/TodoListItem.jsx:2]: Good use of component here. Good to see you do styling also

**Review by LucasIversen — COMMENTED** [review]
Hey Dmytrii. Really good job on your homework! You show a very good understanding of javascript, and the use of components in react, so keep it up 💪

File: react/react1/week1/Todo/src/components/TodoListItem/TodoListItem.jsx (hunk 3 above)
Comment (dmytriigithub) [reply react/react1/week1/Todo/src/components/TodoListItem/TodoListItem.jsx:2]: Hi, Lucas. Thack you for review )

**Pull request merged**